backend/app/routes/admin/changePassword.js

```diff
@@ -6,19 +6,32 @@ const constants = require('../../../constants');
 
 module.exports = async (req, res, next) => {
   const { oldPassword, newPassword } = req.body;
+
   const userRecord = await Admin.findOne({
     email: res.locals.decode.email,
   });
+
   const isPasswordCorrect = await argon2.verify(
     userRecord.passwordHash,
     oldPassword
   );
+
   if (!isPasswordCorrect) {
     const error = new ErrorHandler(constants.ERRORS.INPUT, {
       statusCode: 400,
       message: 'Wrong old password',
       user: userRecord.email,
-      errStack: 'Auth Login',
+      errStack: 'Password change',
+    });
+    return next(error);
+  }
+
+  if (oldPassword === newPassword) {
```

Comment:
this should be the first thing before making DB call to save on latency

Reply:
@jackfrost13 I didn't understand?

Reply:
You should check this before making any DB call

Reply:
ok . so @jackfrost13 after making these changes should I open a new PR?

Reply:
yes, please change api testing also. Then I can review it again if required

Reply:
ok